<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMUpgrade</class>
 <widget class="QWidget" name="SMUpgrade">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="tools" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QToolButton" name="tbShengji">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>4</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbHulue">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>4</y>
      <width>90</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbShaxin">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>4</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>840</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
